index.src.html

```diff
@@ -475,10 +494,11 @@ <h3 dfn export id="referrer-policy-empty-string">The empty string</h3>
 
   <div class="example">
     Given a HTML <{a}> element without any declared <{a/referrerpolicy}>
-    attribute, its referrer policy is "". Thus, navigation requests initiated
-    by clicking on that <{a}> element will be sent with the <a
-    for="Document">referrer policy</a> of the <{a}> element's <a>node
-    document</a>. If that {{Document}} has "" as its referrer policy, the
+    attribute, its referrer policy is "". Thus, navigation requests
+    initiated by clicking on that <{a}> element will be sent with
+    the <a for="Document" lt="document referrer policy">referrer
```

Comment:
I think this was because Document referrer policy was listed as type `dfn` instead of `interface`; I fixed that above and now `{{Document/referrer policy}}` works.